src/dataIntegration/transcriptomics/call_fastcore.m

```diff
@@ -0,0 +1,233 @@
+function tissueModel = call_fastcore(model, expressionRxns, threshold, tol, core, scaling) 

```

Comment:
Completly agree. Note that this tolerance will be directly integrated in the consistency check of the model...